documentation/book/ref-kafka-authorization.adoc

```diff
@@ -25,4 +24,25 @@ To enable simple authorization, the `type` field should be set to `simple`.
 authorization:
   type: simple
 # ...
-----
\ No newline at end of file
+----
+
+== Super users
+
+Super users can access all resources in your Kafka cluster regardless of any access restrictions defined in ACLs.
+To designate super users for a Kafka cluster, enter a list of user principles in the `superUsers` field. If a user has TLS Client Authentication enabled, prefix their user principle with `CN=`.
```

Comment:
What's a 'user principle'?

Reply:
@ppatierno - Please can you check and confirm? I searched for "user principle" in relation to ACLs but could not find anything relevant. Thank you.

Reply:
Technically, the thing prefixed with `CN=` is the common name from the certificate subject. So I would phrase it like this:

```suggestion
To designate super users for a Kafka cluster, enter a list of user principles in the `superUsers` field. 
If a user uses TLS Client Authentication, the username will be the common name from their certificate subject prefixed with `CN=`.
```